### PR TITLE
Minimum Viable Contributing Guide + Readme updates

### PR DESCRIPTION
**Pull request by hthillman**
**What does this pull request do? Explain your changes. (required)**
Updates the readme, adds a contributing guide, and makes some minor updates to documentation structure

**Specific updates (required)**
add CONTRIBUTING.md
update README.md
move changelog instructions to CONTRIBUTING.md

**How did you test each of these updates (required)**
the easiest way to review is probably to jump over to https://github.com/livepeer/go-livepeer/tree/hh-contributing-guide.

**Checklist:**

- [x] Read the [contribution guide](./doc/contributing.md)
- [x] `make` runs successfully
- [x] All tests in `./test.sh` pass
- [x] README and other documentation updated
- [ ] [Pending changelog](./CHANGELOG_PENDING.md) updated

### CODE REVIEW

**Comment by hthillman** [comment]
@yondonfu and/or @iameli can i get a review here? main concerns are probably re: the contribution guide

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+[good first issues](https://github.com/livepeer/go-livepeer/labels/good%20first%20issue).
+
+Many issues are open discussions. Feel free to add your own concerns, ideas, and
+workarounds. If you don't see what you're looking for, you can always open a new
+issue. 
+
+#### 🐛 Fixing bugs, 🕶️ adding feature/enhancements, or 👌 improving code quality
+
+If you're into coding, maybe try fixing a
+[bug](https://github.com/livepeer/go-livepeer/labels/good%20first%20issue)
```
Comment (yondonfu) [inline CONTRIBUTING.md]: Should this link filter for the bug label?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+or taking on a
+[feature request](https://github.com/livepeer/go-livepeer/labels/help%20wanted).
```
Comment (yondonfu) [inline CONTRIBUTING.md]: Should this link filter for the feature request label?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+- **If you are opening an issue...**
+
+  - Fill out all required sections for your issue type. Issues that are not
+    filled out properly will be flagged as `need: more info` and will be closed if not
+    updated.
+  - _Keep your issue simple, clear, and to-the-point_. Most issues do not
+    require many paragraphs of text. In fact, if you write too much, it's
+    difficult to understand what you are actually trying to communicate.
+    **Consider
+    [starting a discussion](https://github.com/livepeer/go-livepeer/issues/new/choose)
```
Comment (yondonfu) [inline CONTRIBUTING.md]: Should this link to the creation option under the "Discussion" tab of the repo?

File: README.md [hunk 4]
```diff
@@ -1,35 +1,71 @@
+<!-- show-on-docup
+<br />
+-->
+
+[![go-livepeer](https://user-images.githubusercontent.com/555740/117340053-78210e80-ae6e-11eb-892c-d98085fe6824.png)](https://github.com/livepeer/go-livepeer)
```
Comment (yondonfu) [inline README.md:5]: Niiiiiiiice

**Review by yondonfu — CHANGES_REQUESTED** [review]
Looks great! Just a few comments about links, but generally this is 👌 👌

**Review by yondonfu — APPROVED** [review]
LGTM! Can you squash these commits before merging?

**Pull request merged**